app/models/form526_submission.rb

```diff
@@ -251,15 +255,13 @@ def birls_id=(value)
   def perform_ancillary_jobs_handler(_status, options)
     submission = Form526Submission.find(options['submission_id'])
     # Only run ancillary jobs if submission succeeded
-    submission.perform_ancillary_jobs(options['first_name']) if submission.jobs_succeeded?
+    submission.perform_ancillary_jobs if submission.jobs_succeeded?
```

Comment:
Don't need to pass `first_name` argument; `get_first_name` can be called when it is needed.